mmv1/products/gkeonprem/VmwareCluster.yaml

```diff
@@ -0,0 +1,662 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "VmwareCluster"
+min_version: beta
+base_url: "projects/{{project}}/locations/{{location}}/vmwareClusters"
+create_url: "projects/{{project}}/locations/{{location}}/vmwareClusters?vmware_cluster_id={{name}}"
+update_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}"
+delete_url: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}?force=true"
+self_link: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}"
+update_verb: :PATCH
+update_mask: true
+description: "A Google VMware User Cluster."
+autogen_async: false
+id_format: "projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}"
+import_format: ["projects/{{project}}/locations/{{location}}/vmwareClusters/{{name}}"]
+taint_resource_on_failed_create: true
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 60
+  delete_minutes: 60
+  update_minutes: 60
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "gkeonprem_vmware_cluster_basic"
+    min_version: beta
+    primary_resource_id: "cluster-basic"
+    vars:
+      name: "basic"
+    test_env_vars:
+      project: "fake-backend-360322"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "gkeonprem_vmware_cluster_f5lb"
+    min_version: beta
+    primary_resource_id: "cluster-f5lb"
+    vars:
+      name: "f5lb"
+    test_env_vars:
+      project: "fake-backend-360322"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "gkeonprem_vmware_cluster_manuallb"
+    min_version: beta
+    primary_resource_id: "cluster-manuallb"
+    vars:
+      name: "manuallb"
+    test_env_vars:
+      project: "fake-backend-360322"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "name"
+    description: The VMware cluster name.
+    immutable: true
+    url_param_only: true
+    required: true
+  - !ruby/object:Api::Type::String
+    name: "location"
+    description: The location of the resource.
+    immutable: true
+    url_param_only: true
+    required: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: "adminClusterMembership"
```

Comment:
Is this mutable? I'd be surprised if so.

Reply:
It is immutable. Thanks for correction!